smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftDependency.kt

```diff
@@ -202,13 +254,20 @@ class SwiftDependency(
     }
 
     private fun toSymbolDependency(): SymbolDependency {
-        return SymbolDependency.builder()
+        val builder = SymbolDependency.builder()
             .putProperty("target", target)
             .putProperty("branch", branch)
             .putProperty("localPath", localPath)
             .packageName(packageName)
             .version(version)
-            .putProperty("url", url)
-            .build()
```

Comment:
If distributionMethod is git then location contains the Git URL.
If it is SPR then location is the package scope.

These properties are written to the SymbolDependency under the appropriate keys.